packages/styling/components/table.scss

```diff
@@ -0,0 +1,329 @@
+@import "../variables";
+@import "../mixins/rich-content";
+
+//== Tables
+//
+//## Customizes the `.table` component with basic values, each used across all table variations.
+
+//** Padding for `<th>`s and `<td>`s.
```

Comment:
deze comment murf mag weg gehaald worden. Controleer ook even of ALLE vars nog in gebruik zijn en die !default mag ook schluss